continuum/scenarios/scenario_utils.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from continuum.datasets import InMemoryDataset
+from continuum.scenarios import ContinualScenario
+
+
+def create_subscenario(base_scenario, task_indexes):
+    """
+    In this function we want to create a subscenario from the different tasks, either by subsampling tasks or reodering
+    or both.
+    """
+
+    new_x, new_y, new_t = None, None, None
+
+    for i, index in enumerate(task_indexes):
+        taskset = base_scenario[index]
+        all_task_indexes = np.arange(len(taskset))
+        x, y, t = taskset.get_raw_samples(all_task_indexes)
+        t = np.ones(len(y)) * i
+        if new_x is None:
+            new_x = x
+            new_y = y
+            new_t = t
+        else:
+            new_x = np.concatenate([new_x, x], axis=0)
+            new_y = np.concatenate([new_y, y], axis=0)
+            new_t = np.concatenate([new_t, t], axis=0)
+    dataset = InMemoryDataset(new_x, new_y, new_t)
```

Comment:
You need to pass to InMemoryDataset several property of the original dataset:

- data_type: already present in the __init__
- bounding_boxes: need to create it in the __init__

Reply:
Done for data_type, subscenario is not yet compatible with bbox